packages/dataviews/src/dataform-controls/radio.tsx

```diff
@@ -7,16 +7,15 @@ import { useCallback } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import type { DataFormControlProps } from '../types';
+import type { DataFormControlProps, WithBulkEditing } from '../types';
 
 export default function Radio< Item >( {
-	data,
 	field,
 	onChange,
 	hideLabelFromVision,
-}: DataFormControlProps< Item > ) {
+	value,
+}: DataFormControlProps< Item > & WithBulkEditing< Item > ) {
```

Comment:
I don't understand this change. maybe I'm missing something when reading the diff though. Is "data" optional? where does "value" come from? I mean where are these type changes defined?

Reply:
I updated the `types.ts` with the above: https://github.com/WordPress/gutenberg/pull/67584/files#diff-05cd2939bbf237a5dbcd116a0da9c9019b05ec1ceaa9d429c5fc872f149f63c4R192-R197

`data` is still passed in, but is either `Item` or `Item | Item[]` depending if combined with `WithBulkEditing< Item >`

Reply:
Do we still need data?

Reply:
Yeah we still need `data` as there are still fields that rely on other data. Like the `parent` field which makes use of the post id and post type ( from the data object ) to filter the parent list. Or the slug field that uses `permalink_template` and `link` properties as well.

Currently I don't display these fields in bulk edit mode, as the logic may change a bit if there are multiple records passed down.
Unless there is a better way to pass these additional values down.